src/Cake.Core.Tests/Fixtures/NotFormattableFixture.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```

Comment:
**nitpick:** it looks like these import statements are unused.

Reply:
this should now be addressed